L1Trigger/L1THGCal/python/hgcalBackendLayer2_fwClustering_cfi.py

```diff
@@ -0,0 +1,132 @@
+        nDistServers1 = cms.uint32(15),
+        distServer1_nIn = cms.uint32(5),
+        distServer1_nOut = cms.uint32(6),
+        distServer1_nInterleave = cms.uint32(4),
+        nCoarsePhiRegionsDist2 = cms.uint32(6),
+        nDistServers2 = cms.uint32(18),
+        distServer2_nIn = cms.uint32(5),
+        distServer2_nOut = cms.uint32(6),
+        distServer2_nInterleave = cms.uint32(4),
+    ),
+
+    # Histogram parameters
+    cColumns=cms.uint32(108),
+    cRows=cms.uint32(44),
+    rOverZHistOffset = cms.uint32(440), # offset of first r/z bin in number of LSB
+    rOverZBinSize = cms.uint32(64), # in number of LSB
+
+    # Smearing parameters
+    smearingParams = cms.PSet(
+        maxBinsSmearing1D=cms.uint32(6),
+        nBitsAreaNormLUT=cms.uint32(17)
+    ),
+
+    # Threshold maxima parameters
+    # Threshold for given histogram row (r/z bin) paramaterized as a+b*bin+c*bin^2
+    thresholdMaximaParams = cms.PSet(
+        a=cms.uint32(18000),
+        b=cms.uint32(800),
+        c=cms.int32(-20)
+    ),
+
+    # Clusterizer parameters
+    clusterizerParams = cms.PSet(
+        nBinsCosLUT = cms.uint32(77), # Number of entries in 1-cos(delta phi) LUT when clustering to seeds
+        nBitsCosLUT = cms.uint32(18), # Number of bits used in 1-cos(delta phi) LUT
+        nFifos=cms.uint32(18),
+        nColumnsPerFifo=cms.uint32(4),
+        firstSeedBin=cms.uint32(18), # First bin considered as a seed for a cluster
+        nColumnsFifoVeto=cms.uint32(6), # Minimum distance between seeds in neighbouring fifos)
+        deltaR2Cut=cms.uint32(20000), # Delta R^2 threshold, in digitised units
+        nColumnsForClustering=cms.uint32(3), # Number of columns either side of column containing seed to stream for clustering
+        nRowsForClustering=cms.uint32(2), # Number of rows either side of row containing seed to consider during clustering
+        clusterizerMagicTime=cms.uint32(434),
+    ),
+    depths = cms.vuint32(0 , # No zero layer
+          0 , 30 , 59 , 89 , 118 , 148 , 178 , 208 , 237 , 267 , 297 , 327 , 356 , 386 , # CE-E
+          415 , 445 , 475 , 505 , 534 , 564 , 594 , 624 , 653 , 683 , 712 , 742 , 772 , 802 , # CE-E
+          911 , 1020 , 1129 , 1238 , 1347 , 1456 , 1565 , 1674 , # CE-FH
+          1783 , 1892 , 2001 , 2110 , 2281 , 2452 , 2623 , 2794 , 2965 , 3136 , 3307 , 3478 , 3649 , 3820 , # CE-BH
+          0 , 0 , 0 , 0 , 0 , 0 , 0 , 0 , 0 , 0 , 0 , 0 , 0 ),
+    triggerLayers = cms.vuint32(0 , # No zero layer
+                  1 , 0 , 2 , 0 , 3 , 0 , # CE-E (early)
+                  4 , 0 , 5 , 0 , 6 , 0 , 7 , 0 , 8 , 0 , # CE-E (core)
+                  9 , 0 , 10 , 0 , 11 , 0 , 12 , 0 , 13 , 0 , 14 , 0 , # CE-E (back)
+                  15 , 16 , 17 , 18 , # CE-H (early)
+                  19 , 20 , 21 , 22 , 23 , 24 , 25 , 26 , 27 , 28 , 29 , 30 , 31 , 32 , 33 , 34 , 35 , 36 , 0 , 0 , # CE-H (back)
+                  0 , 0 , 0 , 0 , 0 , 0 , 0 , 0 , 0 , 0 , 0  ), # Padding
```

Comment:
I think it would be better to derive this from the trigger geometry class, but as you say I don't think the info is accessible in this format (mapping from a layer to a trigger layer).  For now I could modify the trigger geometry class to provide this info (either here or in another PR?).  Longer term I would need to understand if the firmware (and emulator) really needs this info in this particular format or not.

Reply:
That would be great if you can modify the geometry class. 
Rather in an other PR.
Looking quickly into the geometry class, I think adding a getter for `trigger_layers_` could do more or less the job.

Reply:
Done